app/build-js.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
+echo "Setting up path variables and finding node"
 MYPATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-ROOT=$MYPATH/..
-EXTRA_NPM_ARGS="--no-progress --no-spin"
+EXTRA_NPM_ARGS=""
```

Comment:
You're right @lizcross.  We used to do an `npm i` in this file, but it's been moved to the `npm run deps` script. So `EXTRA_NPM_ARGS` can be removed from here safely.  👍